src/java/picard/cmdline/StandardOptionDefinitions.java

```diff
@@ -45,4 +45,5 @@
     public static final String MINIMUM_LOD_SHORT_NAME = "LOD";
     public static final String SORT_ORDER_SHORT_NAME = "SO";
     public static final String USE_ORIGINAL_QUALITIES_SHORT_NAME = "OQ";
+    public static final String PROBABILITY_SHORT_NAME = "P";
```

Comment:
Including this, twice (in downsampleSAM), would be appropriate for "padhopping"-Metric and HiSeqXPFFail as well, I think